keras_hub/src/models/mobilenet/mobilenet_backbone.py

```diff
@@ -458,29 +512,52 @@ def SqueezeAndExcite2D(
             keras.layers.Layer) or keras.activations.Activation instance
             denoting activation to be applied after excite convolution.
             Defaults to `sigmoid`.
+        name: Name of the layer
     """
     if not bottleneck_filters:
         bottleneck_filters = filters // 4
 
-    x = keras.layers.GlobalAveragePooling2D(keepdims=True)(input)
-
+    x = input
     x = keras.layers.Conv2D(
         bottleneck_filters,
         (1, 1),
         data_format=keras.config.image_data_format(),
         activation=squeeze_activation,
+        name=f"{name}_conv_reduce",
     )(x)
     x = keras.layers.Conv2D(
         filters,
         (1, 1),
         data_format=keras.config.image_data_format(),
         activation=excite_activation,
+        name=f"{name}_conv_expand",
     )(x)
 
     x = ops.multiply(x, input)
     return x
 
 
+def ConvBnAct(x, filter, activation, name=None):
```

Comment:
rename ConvBnAct to be more readable -> `conv_batch_norm_activation`.

Please use snake_case for method names